controller/search.js

```diff
@@ -17,6 +17,13 @@ function setup( config, backend, query ){
       return next();
     }
 
+    // do not run controller if there are already results
+    // this was added during libpostal integration.  if the libpostal parse/query
+    // doesn't return anything then fallback to old search-engine-y behavior
+    if (res && res.hasOwnProperty('data') && res.data.length > 0) {
+      return next();
+    }
+
```

Comment:
I added the logging [https://github.com/pelias/api/blob/4032578e29dbb6b0637ebd1b2c23969f39acd059/sanitiser/search_fallback.js#L19-L21](here) since that's the easiest place to determine that there actually was a fallback.